addons/account/models/account_move.py

```diff
@@ -603,6 +603,16 @@ def _sequence_year_range_monthly_regex(self):
         tracking=True,
         help="It indicates that the invoice/payment has been sent or the PDF has been generated.",
     )
+
+    move_sent_state = fields.Selection(
+        selection=[('not_sent', "Not Sent"), ('sent', "Sent")],
+        default='not_sent',
+        tracking=True,
+        copy=False,
+        readonly=True,
+        help="It indicates that the invoice/payment has been sent or the PDF has been generated.",
+    )
```

Comment:
Maybe the spec was not clear enough we actually don't want to get rid of is_move_sent but use it :eyes: With the is_move_sent you can do compute on this field, it will reduce the diff a looot you will see ahah

Reply:
Except that you are in a stable version, so adding a stored selection field would break the database of users :eyes: So i think the best way to handle that in stable is to add a selection field computed base on the is_move_sent also it will reduce the diff by a looot
If we were in master i agree that the field could be replace though